### PR TITLE
Added Gatsby Theme

### PR DESCRIPTION
**Pull request by paolotormon**
Hi, this theme is inspired by the colors of the popular book/movie "The Great Gatsby"

Hope you like it!

anurag-sample

### CODE REVIEW

**Comment by anuraghazra** [comment]
Hi @paolotormon thanks for the theme contribution, this looks good but can you provide any references of where did you get the color codes? because this does not really seems like "gatsby" theme

**Comment by anuraghazra** [comment]
Oh its "The Great Gatsby" okay got it.

File: themes/index.js [hunk 1]
```diff
@@ -119,6 +119,12 @@ const themes = {
     text_color: "FFFFFF",
     bg_color: "050F2C",
   },
+  "gatsby":{
```
Comment (anuraghazra) [inline themes/index.js:122]: maybe the name "gatsby" can also refer to gatsbyjs so i think it'll be better if we change it to something else. great-gatsby?

Comment (paolotormon) [reply themes/index.js:122]: Yes, that change would be great!

**Comment by anuraghazra** [comment]
Thanks @paolotormon for the theme contribution. :)

**Comment by paolotormon** [comment]
> Thanks @paolotormon for the theme contribution. :)
You're welcome @anuraghazra ! 😃

**Pull request merged**